### PR TITLE
LGA-384 uW cron

### PR DESCRIPTION
**Pull request by farrepa**
## What does this pull request do?

Move invocation of `monitor_missing_outcome_codes` script to uW, inside the container
Reconfigure uW to log to stdout
Add Sentry as a log handler, set to error level
Adjust logging in monitor_missing_outcome_codes

## Any other changes that would benefit highlighting?

Use simple formatter for console logging to avoid nested logstash

### CODE REVIEW

File: cla_backend/apps/cla_butler/management/commands/monitor_missing_outcome_codes.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def handle(self, *args, **options):
         if self.should_run_housekeeping(**options):
             self.check_for_missing_outcome_codes()
         else:
-            self.stdout.write('Not doing housekeeping because running on secondary instance')
+            logger.info('Skip check_for_missing_outcome_codes because running on secondary instance')
```
Comment (sldblog) [inline cla_backend/apps/cla_butler/management/commands/monitor_missing_outcome_codes.py]: 👏

File: docker/cla_backend.ini [hunk 2]
```diff
@@ -11,11 +11,9 @@ processes = 2
...
-req-logger = file:/var/log/wsgi/request.log
-logger = file:/var/log/wsgi/error.log
 post-buffering = 1
 harakiri = 20
 buffer-size=32768
 post-buffering-bufsize=32768
-
-cron = 0  5 -1 -1 0 python /home/app/django/manage.py housekeeping
+cron = 0 5 -1 -1 0 python /home/app/django/manage.py housekeeping
+cron = 15 2 -1 -1 -1 python /home/app/django/manage.py monitor_missing_outcome_codes
```
Comment (sldblog) [inline docker/cla_backend.ini]: For posterity, could you link the original location where it was moved from, please?

File: docker/cla_backend.ini [hunk 3]
```diff
@@ -11,11 +11,9 @@ processes = 2
 chdir = /home/app/django
 env = DJANGO_SETTINGS_MODULE=cla_backend.settings
 module = cla_backend.wsgi:application
-req-logger = file:/var/log/wsgi/request.log
-logger = file:/var/log/wsgi/error.log
```
Comment (sldblog) [inline docker/cla_backend.ini]: 👏

File: cla_backend/settings/base.py [hunk 4]
```diff
@@ -266,9 +266,15 @@
         'console': {
             'level': 'INFO',
             'class': 'logging.StreamHandler',
-            'formatter': 'logstash',
+            'formatter': 'simple',
```
Comment (sldblog) [inline cla_backend/settings/base.py]: 🙋‍♂️ It would be great to find a way to avoid nested JSON but still have a JSON outcome. Is that what will happen?

Comment (farrepa) [reply cla_backend/settings/base.py]: Added sample to JIRA 🎟

**Pull request merged**